config/environments/prod.rb

```diff
@@ -13,6 +13,7 @@
   # Full error reports are disabled and caching is turned on.
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
+  config.cache_store = :file_store, "/tmp/rails_cache_store"
```

Comment:
in prod it may be better to use redis or memcache to ensure a single global cache... I'm not sure how many prod web machines we are running right now, or will be soon.

Reply:
I agree that would be best

Reply:
It looks like redis requires some configuration to be used as a cache. See https://guides.rubyonrails.org/caching_with_rails.html#activesupport-cache-rediscachestore . Who is familiar with redis on our team? 

And if we are running a single web machine right now, I don't think it will result in any faster performance, FYI.

Reply:
I think it is required to use redis or memcache, since we currently have 6 instances.

Reply:
please use memcache. we don't need a persitent cache on this. Doing a decent size memcache tier should suffice.

Reply:
Does anybody on the team have experience setting up memcache or redis, ideally for rails cache? 

And do you think we should make that change in this PR?